### PR TITLE
Bug fix/make move cell fcn safe

### LINKED ISSUE #441: move_cell doesn't work correctly
move_cell doesn't work correctly. If the cell is displaced by less than the cell's width than any pixels that were common to both cell locations are set to medium.  Looks like the code is moving the cell to the new location then deleting all pixels from the original location (or setting them to medium) evn if those pixels are part of the cell in its new location.

Also, move_cell requires the PixelTracker plugin but that is not in the manual or in the code snippet that Twedit++ inserts.

Found on WIndows 10, CC3D 4.2.4 20210123

### PR DESCRIPTION
**Pull request by JulianoGianlupi**
fixes #441

### CODE REVIEW

File: cc3d/core/PySteppables.py [hunk 1]
```diff
@@ -1935,7 +1940,10 @@ def move_cell(self, cell, shift_vector):
         # Now we will delete old pixels
         medium_cell = CompuCell.getMediumCell()
         for pixel in pixels_to_delete:
-            self.cell_field[pixel.x, pixel.y, pixel.z] = medium_cell
+			# Safe deletion. Don't delete the old pixel
```
Comment (tjsego) [inline cc3d/core/PySteppables.py:1943]: Indents need reduced.

File: cc3d/core/PySteppables.py [hunk 2]
```diff
@@ -1898,6 +1898,9 @@ def move_cell(self, cell, shift_vector):
         :param shift_vector: {tuple,  list, array, or CompuCell.Point3D} 3-element list specifying shoft vector
         :return: None
         """
+		
+		if not cell:
```
Comment (tjsego) [inline cc3d/core/PySteppables.py:1902]: Indents need reduced.

File: cc3d/core/PySteppables.py [hunk 3]
```diff
@@ -1915,6 +1918,8 @@ def move_cell(self, cell, shift_vector):
         # If we try to reassign pixels in the loop where we iterate over pixel data
         # we will corrupt the container so in the loop below all we will do is to populate the two list mentioned above
         pixel_list = self.get_cell_pixel_list(cell)
+		if pixel_list is None:
```
Comment (tjsego) [inline cc3d/core/PySteppables.py:1921]: Indents need reduced.

**Review by tjsego — CHANGES_REQUESTED** [review]

File: cc3d/core/PySteppables.py (hunk 1 above)
Comment (CompuCell3D) [reply cc3d/core/PySteppables.py:1943]: Thanks @JulianoGianlupi  for looking into this. Let me see how  this behaves in the example that Jim mentioned.

**Review by CompuCell3D — APPROVED** [review]

**Pull request merged**